src/Swarm/Game/Entity.hs

```diff
@@ -313,7 +313,7 @@ buildEntityMap :: [Entity] -> EntityMap
 buildEntityMap es =
   EntityMap
     { entitiesByName = M.fromList . map (view entityName &&& id) $ es
-    , entitiesByCap = M.fromListWith (<>) . concatMap (\e -> map (,[e]) (e ^. entityCapabilities)) $ es
+    , entitiesByCap = M.fromListWith (<>) . concatMap (\e -> map (,[e]) (Set.toList $ e ^. entityCapabilities)) $ es
```

Comment:
I wonder if `entitiesByCap` should change to have type `Map Capability (Set Entity)`?  I don't mean we should change it in this PR, it just made me wonder.